test/unit/app/controllers/network/network-controller-test.js

```diff
@@ -34,8 +34,10 @@ describe('NetworkController', function () {
         assert.equal(providerProxy.test, true)
       })
     })
+
     describe('#getNetworkState', function () {
       it('should return loading when new', function () {
+        networkController = new NetworkController()
```

Comment:
The call to `networkController.initializeProvider(networkControllerProviderConfig)` in `beforeEach` (line 21 above) eventually sets the network state to something other than loading. With `json-rpc-engine@5.2.0`, it does it before we can get the state in the `getNetworkState` test.